### ISSUE TITLE
It's possible login to amazon seller central using Goutte?

### ISSUE THREAD

**Issue by linkinkin** [FriendsOfPHP/Goutte#272]
Hi,

Is there a way to login to seller central using Goutte?
I tried this but it keeping saying password wrong (I tried manually and it works fine):

```
$client = new Client();
$crawler = $client->request('GET', 'https://sellercentral.amazon.com');
$form = $crawler->selectButton('sign-in-button')->form();
$crawler = $client->submit($form, array('username' => 'test@test.com', 'password' => '123456'));

```
Thanks

**Comment by aik099** [comment]
If you can do it in Web Browser with JavaScript disabled, then it can be done via Goutte as well.

**Comment by linkinkin** [comment]
Thanks for your reply.

Yes, I can with javascript disabled but not using goutte.
Actually I just figure out what is the problem. The form has one hidden input that must be set the value to "1". Goutte don't show me this element when I `$form->getValues()` maybe because this element doesn't have the atribute name set (only id). Any idea how I can set this hidden input to "1"?

Regards

**Comment by aik099** [comment]
> maybe because this element doesn't have the atribute name set (only id).

If element doesn't have `name` attribute, then it doesn't go into form submit and therefore can't affect how data is being processed by server.

> Any idea how I can set this hidden input to "1"?

Without `name` attribute element isn't considered a form field I guess and therefore you need to use crawler to find it and change it's contents.

**Comment by linkinkin** [comment]
> If element doesn't have name attribute, then it doesn't go into form submit and therefore can't affect how data is being processed by server.

Yes, you're right, I changed manually and the error persist.

After I submit the form I can just `echo $crawler->html();` to get the resulted page right?

Thanks

**Comment by aik099** [comment]
> After I submit the form I can just echo $crawler->html(); to get the resulted page right?

You'd better try that first and then ask if that doesn't work. Also I bet this is documented somewhere.

**Comment by linkinkin** [comment]
Yes, off course I tried, I just want to figure out what is not working. Anyways, it seems that Goutte will not help me with that. Have to find another way. Thanks!

**Comment by NanheKumar** [comment]
@linkinkin Can You suggest what is best way because i am struggling with same issue.
